js/components/communityresource/list.vue

```diff
@@ -22,7 +22,13 @@ export default {
                 return new CommunityResources();
             }
         },
-        withoutDataset: Boolean
+        withoutDataset: Boolean,
+        title: {
+            type: String,
+            default: function() {
```

Comment:
Some code below is converted to ES6 syntax. Why not `default() {…}` here and in the rest?